<launch>

  <node name="image_sub_node"  pkg="rr_auto_dock" type="image_sub_node" output="screen"/>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video-front" />
    <param name="camera_frame_id" value="front_camera_link" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="framerate" value="30" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_info_url" value="package://rr_auto_dock/misc/ost.yaml" />
    <remap from="/usb_cam/teleop_cam" to="/camera"/>
    <remap from="/usb_cam/camera_info" to="/camera_info"/>
  </node>

</launch>